homeassistant/components/remote/strings.json

```diff
@@ -38,11 +38,11 @@
     },
     "toggle": {
       "name": "[%key:common::action::toggle%]",
-      "description": "Toggles a device on/off."
+      "description": "Sends the toggle command to a device to turn it on or off."
```

Comment:
In theory we don't send the toggle command, we send the on or off command based on the state

Reply:
OK, no problem, how about:

    Sends the turn on or turn off command to toggle a device.